src/main/java/org/jabref/gui/fieldeditors/contextmenu/DefaultMenu.java

```diff
@@ -62,6 +62,7 @@ private static Menu getConversionMenu(TextInputControl textInputControl) {
             CustomMenuItem menuItem = new CustomMenuItem(new Label(converter.getName()));
             Tooltip toolTip = new Tooltip(converter.getDescription());
             Tooltip.install(menuItem.getContent(), toolTip);
+            menuItem.setDisable(textInputControl.textProperty().get() == null);
```

Comment:
This will only disable the menu items at initialization. For instance, if you type something into the text field and delete it, the menu items aren't disabled. You need to call `menuItem.setDisable(textInputControl.textProperty().get() == null);` whenever textInputProperty changes. There is an elegant solution for this using `EasyBind#subscribe`. Also, you need to handle the case when the text is empty. We already have a method for that; see `StringUtil#isNullOrEmpty`.